### ISSUE TITLE
CD Command Response Time

### ISSUE THREAD

**Issue by aliemre** [mdkholy/dashbrew#42]
After vagrant ssh, during setting the directory by using cd command, if the project has a git vcs, cd command response time is nearly 4-5 s. If anyone has faced with the same issue?

**Comment by nickashley** [comment]
Yes, I have had this issue as well. Deleting the dotfiles in /config/home speeds things up, if you don't care about the command-line feature enhancements
